ddtrace/tracer/tracer.go

```diff
@@ -675,3 +702,17 @@ func (t *tracer) hostname() string {
 	}
 	return hostname.Get()
 }
+
+func PrintOpenSpans() {
+	t, ok := internal.GetGlobalTracer().(*tracer)
+
+	if !ok {
+		log.Warn("Could not find active tracer")
+		return
+	}
+	if !t.config.debugOpenSpans {
+		log.Warn("Debugging open spans is not enabled")
+		return
+	}
+	t.cPrint <- struct{}{}
+}
```

Comment:
My opinion would be to omit this public method for now, unless customers are asking for it. The more important functionality is that we automatically log the abandoned spans at some useful interval to customers. Providing the API this way would require some code and perhaps a go routine on the customer side to print this with some regular interval, which we can just do for them.

Reply:
I think that makes sense! @knusbaum, @ajgajg1134 , and I were talking about it, and the idea was more to provide this as an option for testing for the user (without turning on `WithDebugSpansMode()`). But maybe the value of this function is already taken care of with the regular logging.

Reply:
Ah it's in the `case <-t.stop:`. Great, thanks!